isminet/models/system.py

```
@@ -0,0 +1,156 @@
+        
+        Raises:
+            ValueError: If the status is not one of 'ok', 'warning', or 'error'.
+        
+        Example:
+            validate_status('ok')  # Returns 'ok'
+            validate_status('warning')  # Returns 'warning'
+            validate_status('invalid')  # Raises ValueError
+        """
+        valid_statuses = {"ok", "warning", "error"}
+        if v not in valid_statuses:
+            raise ValueError(f"Status must be one of: {', '.join(valid_statuses)}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)